### PR TITLE
ZIP-50: otherpubs endpoint (zoom schedule) now requires auth

### PR DESCRIPTION
**Pull request by lbjay**

### CODE REVIEW

File: functions/zoom-uploader.py [hunk 1]
```diff
@@ -37,8 +37,11 @@
 session.auth = HTTPDigestAuth(OPENCAST_API_USER, OPENCAST_API_PASSWORD)
 session.headers.update({
     "X-REQUESTED-AUTH": "Digest",
+    # TODO: it is possible this header is not actually necessary for the
+    #  endpoints being used here. It seems like for Opencast endpoints where
+    # it is necessary the correct header is actually
+    # "X-Opencast-Matterhorn-Authorizatoin"
```
Comment (nataliemona) [inline functions/zoom-uploader.py]: typo

File: tasks.py [hunk 2]
```diff
@@ -626,9 +627,19 @@ def import_dce_schedule_from_opencast(ctx, endpoint=None):
     if endpoint is None:
         engage_host = oc_host(ctx, 'engage')
         endpoint = "https://{}/otherpubs/search/series.json".format(engage_host)
+
+    session = requests.Session()
+    api_user = getenv('OPENCAST_API_USER')
```
Comment (nataliemona) [inline tasks.py]: trying to be consistent about using double quotes

**Review by nataliemona — APPROVED** [review]

**Comment by rute-santos** [comment]
Same comments as Mona. Approved.

**Review by rute-santos — APPROVED** [review]

File: functions/zoom-uploader.py (hunk 1 above)
Comment (lbjay) [reply functions/zoom-uploader.py]: ugh cursed header

**Pull request merged**